### DISCUSSION TITLE
Insects model

### DISCUSSION THREAD

**Discussion by assa83** [birdnet-team/BirdNET-Analyzer#310] [General]
Dear all

I am testing the library with Insects sounds and it seems there is no recognition of these sounds. Am I doing something wrong or the library is focused purely on birds?

Another question, is there a library specific for insects?

Regards
Mattia

**Comment by euxoa** [comment]
It's focused on birds, with some classes of other sounds added to the model to keep them not interfering with bird-sound recognition. 

My guess is representations in the model are quite good for insect sounds however, except maybe highest frequencies. So it may be possible to fine tune the model to recognize insects, if you have labelled training material.
